packaging/ubuntu-16.04/rules

```diff
@@ -54,7 +54,7 @@ ifeq ($(shell dpkg-vendor --query Vendor),Ubuntu)
     # libudev and udevd so we need to link with it dynamically.
     VENDOR_ARGS=--enable-nvidia-ubuntu --enable-static-libcap --enable-static-libapparmor --enable-static-libseccomp
 else
-    VENDOR_ARGS=--disable-apparmor
+    VENDOR_ARGS=--disable-apparmor --disable-seccomp
```

Comment:
@zyga You mean a second ifeq within the else part or dropping the else part completely?